pkg/config/toml/defaults/Ethereum_Mainnet.toml

```diff
@@ -4,6 +4,8 @@ LinkContractAddress = '0x514910771AF9Ca656af840dff83E8264EcF986CA'
 MinContractPayment = '0.1 link'
 OperatorFactoryAddress = '0x3E64Cd889482443324F91bFA9c84fE72A511f48A'
 NoNewFinalizedHeadsThreshold = '9m'
+TxMaximumWaitTimeForConfirmation = '12s'
```

Comment:
I think these values are reversed

Reply:
You are right. I'll make the minimum time 12 seconds which is what takes to create a block and give 20 seconds to be safe that we have enough time before failing.

Reply:
Fixed.